style.css

```diff
@@ -0,0 +1,695 @@
+    justify-content: space-between;
+    width: 92%;
+    height: 50%;
+    margin-top: 2%;
+}
+
+.section-about > p {
+    font-weight: 700;
+    font-size: 200%;
+}
+
+.section-about p {
+    display: inline-block;
+    padding-left: 1%;
+    color: #ffffff;
+    margin-top: 1em;
+    color: purple;
+}
+
+.section-about-me-left {
+    padding: 10px 0;
+    margin: 0;
+}
+
+/* about me let side */
+.section-about-me-left {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.section-about-me-left > img {
+    width: 12em;
+    height: 12em;
+    margin: 1em;
+    border-radius: 50%;
+    display: flex;
+    align-self: center;
+}
+
+.section-about-me-left-ul {
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    align-items: center;
+    width: 50%;
+    margin: auto 0;
+}
+
+.section-about-me-left-ul li {
+    list-style: none;
+    font-size: 1.5em;
+    color: purple;
+    padding: .8rem;
+}
+
+.section-about-me-left-ul li span {
+    color: rgb(36, 36, 36);
+    font-weight: 700;
+}
+
+/* about me right */
+.section-about-me-right {
+    padding: 0 1rem 1rem 1rem;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.04em;
+    width: 50%;
+}
+
+/* text about me (right side) */
+.section-about-me-right p {
+    font-size: 1.2em;
+    padding: 1em;
+    text-align: justify;
+    border-left: .4rem dotted purple;
+}
+
+/* skills*/
+.section-skills {
+    float: right;
+    background-color: #fafaff;
+    padding: 3em;
+    margin-top: 2%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: baseline;
+    width: 85%;
+    align-self: flex-end;
+    color: purple;
+    height: 50%;
+}
+
+.section-skills-left {
+    border-right: .4rem dotted rgb(143, 77, 143);
+    padding: 1em;
+    text-align: center;
+    width: 32%;
+    font-size: 1.5rem;
+    font-weight: 700;
+}
+
+.section-skills-center {
+    border-right: .4rem dotted rgb(143, 77, 143);
+    padding: 1em;
+    text-align: center;
+    width: 32%;
+    font-size: 1.5rem;
+    font-weight: 700;
+}
+
+.section-skills-right {
+    width: 32%;
+    padding: 1em;
+    text-align: center;
+    font-size: 1.5rem;
+    font-weight: 700;
+}
+
+.section-skills img {
+    height: 100px;
+}
+
+/*-------------------------------------------------*/
+/* section gallery (Portfolio) */
+/*-------------------------------------------------*/
+
+
+.section-Portfolio {
+    margin: 3% auto;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+}
+
+.section-Portfolio p {
+    margin-bottom: 2rem;
+    display: inline-block;
+    padding-left: 1%;
+    font-size: 200%;
+    color: #ffffff;
+    font-weight: 700;
+    color: purple;
+}
+
+.section-Portfolio-gallery {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.section-Portfolio-img {
+    align-self: center;
+    opacity: .7;
+    bottom: 0%;
```

Comment:
Jeśli używamy miary 0, to nie ważne jaka to jednostka, i tak to zawsze 0, czyli nic :) Możemy wtedy pomijać zapis jednostki.